doc/cephadm/troubleshooting.rst

```diff
@@ -176,3 +176,39 @@ To access the admin socket, first enter the daemon container on the host::
 
     [root@mon1 ~]# cephadm enter --name <daemon-name>
     [ceph: root@mon1 /]# ceph --admin-daemon /var/run/ceph/ceph-<daemon-name>.asok config show
+
+Manually deploying a MGR daemon
+-------------------------------
+
+cephadm requires a MGR daemon in order to manage the cluster. Please
+follow these steps in order to deploy a MGR ``mgr.hostname.smfvfd`` manually.
+
+Disable the cephadm scheduler, in order to prevent cephadm from removing the new 
+MGR::
+
+  ceph config set mgr mgr/cephadm/pause true
+
+Then get or create the auth entry for the new MGR::
+
+  ceph auth get-or-create mgr.hostname.smfvfd mon "profile mgr" osd "allow *" mds "allow *"
+
+Get the ceph.conf::
+
+  ceph config generate-minimal-conf
+
+Get the container image::
+
+  ceph config get "mgr.hostname.smfvfd" container_image
+
+Create a file ``config-json.json``:
```

Comment:
The content of the file is not seen in the file preview:

Create a file ``config-json.json``::